Issues with example chat

First seems to be some change in czmq

```
chat.c: In function ‘main’:
chat.c:85:9: error: too many arguments to function ‘zstr_send’
         zstr_send (chat_pipe, "%s:%s", argv [1], message);
         ^
```

However when I just remove the message construction and just pass message. I'm getting some weird errors.
I have 2 chat client nodes. If one sends "hello" the other receives this:

```
--------------------------------------
[005] SHOUT
[032] 45567214894743DDBCA9B52EB8688AEB
[004] CHAT
[004] CHAT
[006] hello
```

Notice the double "[004] CHAT"
If the receiving node then sends "hello back" in return:

```
hello back
CHAT
```

notice the CHAT message. I don't think it belongs there

The receiving node als receives the double CHAT:

```
--------------------------------------
[005] SHOUT
[032] DE8978FA2409419F838757DD0A592AD6
[004] CHAT
[004] CHAT
[011] hello back
```

I haven't dug around the code yet but perhaps someone instantly recognizes this?

--- a/examples/chat/chat.c
+++ b/examples/chat/chat.c
@@ -82,7 +82,7 @@ int main (int argn, char *argv [])
         char message [1024];
         if (!fgets (message, 1024, stdin))
             break;
-        zstr_send (chat_pipe, "%s:%s", argv [1], message);
+        zstr_sendf (chat_pipe, "%s:%s", argv [1], message);
     }
     zctx_destroy (&ctx);
     return 0;

